.github/workflows/stale.yml

```diff
@@ -38,6 +38,8 @@ jobs:
         stale-pr-message: 'Stale pull request message'
         stale-issue-label: 'no-issue-activity'
         exempt-issue-labels: 'priority,bug,good first issue'
+        exempt-issue-milestones: 'Structured results'
+        stale-pr-label: 'no-pr-activity'
```

Comment:
the `stale-pr-label` line is duplicated. 

do we need separate labels for stale issues and stale PRs? should we just have one? The default value for both (if the parameter isn't provided) is just `Stale`

Reply:
I removed the duplicated `stale-pr-label`. 

If I want to change to the default label for both stale issues and stale PRs, do I remove the entire line or just the defined name?

I also updated the stale PR message to match the stale issue message. Should I change the `days-before-pr-close` value to -1 as well so that PRs are not automatically closed either?